diff --git a/src/meson.build b/src/meson.build
index 63f9d53..605d049 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -216,16 +216,6 @@ sources += configure_file(
   configuration: conf_internal
 )
 
-version_h = vcs_tag(
-  command: ['git', 'describe', '--dirty'],
-  fallback: version_pretty,
-  replace_string: '@buildver@',
-  input: 'version.h.in',
-  output: 'version.h',
-)
-
-sources += version_h
-
 if host_machine.system() == 'windows'
   windows = import('windows')
   sources += windows.compile_resources(libplacebo_rc, depends: version_h,
diff --git a/src/version.h b/src/version.h
new file mode 100644
index 0000000..1f165ea
--- /dev/null
+++ b/src/version.h
@@ -0,0 +1 @@
+#define BUILD_VERSION "vacon built-in"
